src/changelog/buildpacks/_posts/2023-09-18-php-composer-2.6.3.md

```diff
@@ -0,0 +1,9 @@
+---
+modified_at: 2023-09-18 16:00:00
+title: 'PHP - Release Composer version 2.6.3'
+github: 'https://github.com/Scalingo/php-buildpack'
+---
+
+Changelog:
+
+* [Composer 2.6.3](https://github.com/composer/composer/releases/tag/2.6.3)
```

Comment:
I would add the release notes for Composer from 2.6.0 to 2.6.3 as it seems to be the first 2.6.x release

Reply:
In fact it's already been documented in the changelog for `2.6.2`, which was released on Sep. the 5th (https://doc.scalingo.com/changelog).

For some reason our documentation hasn't been updated that day :(